android/build.gradle

```diff
@@ -34,6 +34,12 @@ def getExtOrIntegerDefault(name) {
 }
 
 android {
+    def agpVersion = com.android.Version.ANDROID_GRADLE_PLUGIN_VERSION
```

Comment:
@vikrantnegi  Can you import `com.android.Version` and then replace it with `Version.ANDROID_GRADLE_PLUGIN_VERSION`

Reply:
Done